src/main/main.c

```diff
@@ -1370,8 +1376,8 @@ int re_thread_check(void)
  *
  * @note only used by tmr module
  */
-struct list *tmrl_get(void);
-struct list *tmrl_get(void)
+struct tmrl *tmrl_get(void);
+struct tmrl *tmrl_get(void)
```

Comment:
I suggest to make this api public. Then the application can poll the timers
without starting re_main loop.